src/cc_helpers.cc

```diff
@@ -242,7 +242,14 @@ bool certifier::framework::cc_trust_data::initialize_keystone_enclave_data(const
       printf("initialize_keystone_enclave_data: Not a simulated enclave\n");
       return false;
     }
-    // Todo
+    int m_size = file_size(measurement_file_name);
+    if (m_size < 0) {
```

Comment:
This was just not being checked. This missing check caused me tons of time wasted when porting the Keystone-shim based sample app to run under `run_example.sh` .

The generated measurement file was named slightly differently than the default name used by other apps; i.e. `example_app.measurement`. The initial instructions had named it `keystone_example_app.measurement`, which caused the file to not-be-found and some parts of the workflow would silently exit.

This new error message will improve the situation in future when others attempt to use the workflow and make a similar mistake.